src/api/app/views/webui2/webui/user/_involvement.html.haml

```diff
@@ -71,8 +71,5 @@
                     = link_to(project_name, project_show_path(project: project_name))
 
 = content_for :ready_function do
-  :plain
-    $('#involved-packages-table, #involved-projects-table, #owned-table').DataTable({
-      language: { search: '', searchPlaceholder: "Search..." }
-    });
+  initializeDataTable('#involved-packages-table, #involved-projects-table, #owned-table');
```

Comment:
The function has 2 parameters, but 3 are passed here. I guess this is the issue.

Reply:
no, I am passing one and the second is `null`... I'll :mag:

Reply:
Oh, I missed the quote, sorry!